source/github-helpers/dom-formatters.tsx

```diff
@@ -20,10 +20,10 @@ export const codeElementsSelector = [
 ];
 
 export function shortenLink(link: HTMLAnchorElement): void {
-	// Exclude the link if the closest element found is not `.comment-body`
+	// Exclude the link if the closest element found is not `.markdown-body`
 	// This avoids shortening links in code and code suggestions, but still shortens them in review comments
 	// https://github.com/refined-github/refined-github/pull/4759#discussion_r702460890
-	if (link.closest(String([...codeElementsSelector, '.comment-body']))?.classList.contains('comment-body')) {
+	if (link.closest(String([...codeElementsSelector, '.markdown-body']))?.classList.contains('markdown-body')) {
```

Comment:
I confirmed that by removing both this line _and_ `linkifiedURLClass` the links in https://github.com/refined-github/sandbox/pull/98 are (incorrectly) shortened, which means that this class works on the old version too.

I had broken `linkify-code` earlier which made it difficult to test.

- #7985